ci/build-rpms.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+rm -rf build
```

Comment:
Do we need to use different output dir name? `build` is a bit confusing, we are using this name for everything.

I don't know, please give some insights on this.

Reply:
I'd like to keep the top dir clean. `build` is used by the CI build too. Here it's actually only used to call `cmake` to fill in the variables in the `*.in` files needed.